### PR TITLE
get must-gather image from imagestream

### PR DESCRIPTION
**Pull request by sanchezl**
User the `must-gather` ImageStream in the `openshift` namespace to get the image for the `must-gather` tool.

### CODE REVIEW

**Comment by sttts** [comment]
#22430 merged.

/hold cancel

File: pkg/oc/cli/admin/mustgather/mustgather.go [hunk 1]
```diff
@@ -20,6 +20,9 @@ import (
 	"k8s.io/kubernetes/pkg/kubectl/scheme"
 	"k8s.io/kubernetes/pkg/kubectl/util/templates"
 
+	"github.com/openshift/client-go/image/clientset/versioned/typed/image/v1"
+
```
Comment (deads2k) [inline pkg/oc/cli/admin/mustgather/mustgather.go]: nit, no space

**Comment by deads2k** [comment]
@sanchezl needs an e2e test before we can merge this

**Comment by deads2k** [comment]
now, let's see if this works

/lgtm

**Pull request merged**